spec/migrations/update-6_0_0/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Rule, SchematicContext, Tree, SchematicsException, chain,  } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export function rxjsV6MigrationSchematic(_options: any): Rule {
+  return chain([
+    addDependencies()
+  ]);
+}
+
+const rxjsCompatVersion = '^6.0.0-rc.0';
+const rxjsTSLintVersion = '0.0.0';
+
+function addDependencies() {
+  return (tree: Tree, context: SchematicContext) => {
+      const pkgPath = '/package.json';
+      const buffer = tree.read(pkgPath);
+      if (buffer == null) {
+        throw new SchematicsException('Could not read package.json');
+      }
+      const content = buffer.toString();
+      const pkg = JSON.parse(content);
+
+      if (pkg === null || typeof pkg !== 'object' || Array.isArray(pkg)) {
+        throw new SchematicsException('Error reading package.json');
+      }
+
+      if (!pkg.dependencies) {
+        pkg.dependencies = {};
+      }
+      if (!pkg.devDependencies) {
+        pkg.devDependencies = {};
+      }
+
+      pkg.dependencies['rxjs-compat'] = rxjsCompatVersion;
+      pkg.devDependencies['rxjs-tslint'] = rxjsTSLintVersion;
```

Comment:
Can we skip this install for now? Since we 3ant people to primarily update their app to v6 we shouldn't bring deps they absolutely don't need. Because if we do they will not know when it's ok remove them.